partner_stage/models/res_partner.py

```diff
@@ -14,7 +14,7 @@ def _get_default_stage_id(self):
         )
 
     @api.model
-    def _read_group_stage_id(self, states, domain, order):
+    def _read_group_stage_id(self, states, domain):
         return states.search([])
```

Comment:
Is this supposed to change to
```python
        return states.search([], order=order)
```
otherwise I don't understand what this pull request is attempting. Unit tests could be illuminating.

Reply:
This seems to be fixing the wrong method argument cause the error #2055